src/app/Plans/PlansPage.tsx

```diff
@@ -1,14 +1,125 @@
 import * as React from 'react';
-import { PageSection, Title } from '@patternfly/react-core';
-
-const PlansPage: React.FunctionComponent = () => (
-  <>
-    <PageSection>
-      <Title headingLevel="h1" size="lg">
-        Plans Page Title
-      </Title>
-    </PageSection>
-  </>
-);
+import {
+  Card,
+  PageSection,
+  CardBody,
+  EmptyState,
+  EmptyStateIcon,
+  Title,
+  Button,
+  Bullseye,
+  Spinner,
+  Tooltip,
+} from '@patternfly/react-core';
+import { AddCircleOIcon } from '@patternfly/react-icons';
+import spacing from '@patternfly/react-styles/css/utilities/Spacing/spacing';
+import PlansTable from './components/PlansTable';
+import { IAddPlanDisabledObjModel } from './types';
+
+// TODO replace these with real state e.g. from redux
+import { MOCK_PLANS } from './mocks/plans.mock';
+import { MOCK_PROVIDERS } from '@app/Providers/mocks/providers.mock';
+const IsFetchingInitialPlans = false; // Fetching for the first time, not polling
+const migplans = MOCK_PLANS;
+const providers = MOCK_PROVIDERS;
+
+const PlansPage: React.FunctionComponent = () => {
+  const VMWareList = providers.map((x) => x.spec.type === 'vsphere');
```

Comment:
The capitalization of `VMWareList` and `CNVList` here is odd for an array of data. I think we should reserve capitalized names for modules like components and types. So these could be `vmwareList` and `cnvList`.

Also you can `import { ProviderType } from '@app/common/constants';` and use `ProviderType.vsphere` and `ProviderType.cnv` instead of the plain strings, if you want. Although I think TS validates those either way because of the type of `providers`.

Reply:
Thanks for `@app/common/constants` pointer.